<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCalculator</class>
 <widget class="QDialog" name="CCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCalculator</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="9">
    <widget class="QListView" name="values"/>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="btn_7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="btn_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="btn_8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="btn_5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="btn_6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="btn_9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="btn_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="btn_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="btn_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QToolButton" name="btn_period">
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="btn_plus">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QToolButton" name="btn_enter">
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="btnMinus">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QToolButton" name="btn_0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QToolButton" name="btn_CA">
     <property name="text">
      <string>Clr All</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="btn_Average">
     <property name="text">
      <string>Average</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>values</tabstop>
  <tabstop>btn_7</tabstop>
  <tabstop>btn_8</tabstop>
  <tabstop>btn_9</tabstop>
  <tabstop>btn_4</tabstop>
  <tabstop>btn_5</tabstop>
  <tabstop>btn_6</tabstop>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_3</tabstop>
  <tabstop>btn_period</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
